src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1006,6 +1007,52 @@ public void deleteOrganization(long id) {
                 handleList(Organization.class, "organizations"));
     }
 
+    public Iterable<OrganizationMembership> getOrganizationMemberships() {
+        return new PagedIterable<OrganizationMembership>(cnst("/organization_memberships.json"),
+                handleList(OrganizationMembership.class, "organization_memberships"));
+    }
+
+    /**
+     * Use either the user id or organization id, and set the other to a value < 0. Will only search based
+     * on user id if the user id is > 0.
+     *
+     * @param user_id
+     * @param organization_id
+     * @return
+     */
+    public Iterable<OrganizationMembership> getOrganizationMemberships(long user_id, long organization_id) {
```

Comment:
Oops, I thought both parameters were needed for user_id based lookup. Then, it would be better to have 2 different methods called something like `getOrganizationMembershipsForUser` and `getOrganizationMembershipsForOrg`. WYDT?